lib/Fenix/Model/IRC.pm

```diff
@@ -44,6 +45,9 @@ has handlers => sub($self) {
         } keys %{$self->handler}
     ]
 };
+has sqlite   => sub($self) {
+    Mojo::SQLite->new->from_filename(Mojo::File->new($self->datadir, 'irclog.db'));
```

Comment:
if you want so set any pragmas on the sqllite db than something like this is handy

```suggestion
    my $sql = Mojo::SQLite->new->from_filename(Mojo::File->new($self->datadir, 'irclog.db'));
    
    #$sql->on(connection => sub ($sql, $dbh) {
    #    $dbh->do('PRAGMA foreign_keys = ON;');
    #});
    return $sql
```